### ISSUE TITLE
Make it easier to go to a specific user's profile

### ISSUE THREAD

**Issue by Powersource** [fraction/oasis#359]
## What's the problem you want solved?

If I want to visit someone specific's profile then I currently have to

1. go to the search page
2. search for their name
3. look through the search results for where the user is mentioned, and click on them

## Is there a solution you'd like to recommend?

Tbh using oasis I'm really missing patchwork's search bar (including the dropdown-thing/autocompletion), it makes navigation between users/channels/searches really fast and easy.

Probably a bit related to https://github.com/fraction/oasis/issues/47

Maybe a solution until we've figured that stuff out would be to show matching user(names) at the top of the search results.

**Comment by christianbundy** [comment]
Yes! Thanks for pointing this out.

Maybe a good first step would be adding the search bar to the sidebar/header navigation? Then a next step might be adding all friends to a [`<datalist>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist). Hashtags would be great too.

**Comment by Powersource** [comment]
Yes moving the search bar to the sidebar/header would be a good first step :)
